core/templates/dev/head/components/summary_tile/collection_summary_tile_directive.html

```diff
@@ -2,7 +2,7 @@
   <md-card class="oppia-activity-summary-tile">
     <a ng-href="<[getCollectionLink()]>" style="text-decoration: none;">
       <div class="title-section" style="background-color: <[getThumbnailBgColor()]>;">
-        <img class="collection-corner-image" ng-src="/assets/images/general/collection_corner.svg" alt="<[getCategory()]>">
+        <img class="collection-corner-image" ng-src="/assets/images/general/collection_corner.svg">
```

Comment:
I get that you removed the misplaced attribute, however, this image should have an empty alt attribute to be accessible. So please return that.